test/src/main/java/org/apache/accumulo/test/TestMultiTableIngest.java

```diff
@@ -95,8 +95,7 @@ public static void main(String[] args) throws Exception {
         // populate
         for (int i = 0; i < opts.count; i++) {
           Mutation m = new Mutation(new Text(String.format("%06d", i)));
-          m.put(new Text("col" + Integer.toString((i % 3) + 1)), new Text("qual"),
-              new Value("junk"));
+          m.put(new Text("col" + ((i % 3) + 1)), new Text("qual"), new Value("junk"));
```

Comment:
I think there's a CharSequence version of this:

```suggestion
          m.put("col" + ((i % 3) + 1), "qual", "junk");
```